openedx_webhooks/tasks/github.py

```diff
@@ -85,6 +89,14 @@ def pull_request_opened(self, pull_request, ignore_internal=True, check_contract
         # not an open source pull request, don't create an issue for it
         msg = "@{user} opened PR #{num} against {repo} (internal PR)".format(user=user, repo=repo, num=num)
         log_info(self.request, msg)
+        # new release candidate for Open edX is available, ask internal PR if should be cherry picked
+        if open_edx_release:
+            github_prepare_cherry_pick_comment(github, pr, open_edx_release)
+            return None, True
+        elif test_open_edx_release:
+            if user in ['mduboseedx']:
+                github_prepare_cherry_pick_comment(github, pr, open_edx_release)
```

Comment:
This line is the same as 94.  Two things about that: 1) `open_edx_release` will be None here, so it won't work, and 2) ideally the action line will occur only once, conditionalized on a `doit` boolean that is computed based on the earlier conditions.  I suppose putting these two ideas together, we could keep the action lines separate so they could have different third arguments?